packages/mattermost-redux/src/selectors/entities/threads.ts

```diff
@@ -68,19 +67,12 @@ export function getThreads(state: GlobalState): IDMappedObjects<UserThread> {
 }
 
 export function getThread(state: GlobalState, threadId?: $ID<UserThread>) {
-    const threads = getThreads(state);
-    const myChannels = getMyChannels(state).map((c) => c.id);
-
     if (!threadId) {
         return null;
     }
 
-    const thread = threads[threadId];
-    if (!thread || (thread.post && myChannels.indexOf(thread.post.channel_id) === -1)) {
-        return null;
-    }
-
-    return thread;
```

Comment:
@calebroseland I'll be removing the guard on threads.
 I know we discussed this in the past (https://github.com/mattermost/mattermost-webapp/pull/8793#discussion_r709256045) but this guard creates more problems than it solves for us. 

Do you maybe have a way to reproduce the case of a thread from another team is going to appear in another team's global threads?